blueprints/ember-cli-notifications/index.js

```diff
@@ -7,9 +7,11 @@ module.exports = {
   // Use config to determine whether Font Awesome is imported into consuming app
   afterInstall: function(app) {
     var projectConfig = this.project.config(app.env);
-    var config = projectConfig['ember-cli-notifications'];
+    var config = projectConfig['ember-cli-notifications'] ||
+                 app.options['ember-cli-notifications'] ||
```

Comment:
I'll update this. My thinking here was that most addons configure themselves in `ember-cli-build.js` and not in `environment.js`, specially for stuff that has to do with the build process. Selecting which libs to include is part of that process. But then, on the other hand, it's hard to get those values in the component if defined there. Maybe that change belongs to a different PR.